test/aktivitetskrav/VurderAktivitetskravTest.tsx

```diff
@@ -211,7 +224,8 @@ describe("VurderAktivitetskrav", () => {
       renderVurderAktivitetskrav(aktivitetskrav, oppfolgingstilfelle);
 
       clickButton(buttonTexts["AVVENT"]);
-      clickButton("Lagre");
+      const lagreButtons = screen.getAllByRole("button", { name: "Lagre" });
+      lagreButtons[1].click();
```

Comment:
Nå som modalen for `Avvent` og `Ikke aktuell` kommer "over/oppå" en av tab-sidene som er åpne, så vil man finne to `Lagre`-knapper i DOMen. Dette er én måte å løse det på, men det hadde vært enda bedre om man feks kunne brukt `within` på en eller annen måte. Av eksempler jeg fant i koden og [på nett](https://testing-library.com/docs/dom-testing-library/api-within/), så er `within` bare etterfulgt av et `getSomething...`. Hadde vært penest om man feks kunne skrevet eller lignende:
```typescript
within(modal, () => {
	clickButton("Lagre")
})
```

Noen råd?

Reply:
Funker det å hente ut riktig knapp med `within.getByRole()` og så gjøre `userEvent.click()` på den?

Reply:
Ja, funka med `const lagreButton = within.getByRole(modal).getByRole(button)`! 🎉